starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,9 @@
+----
+<ul>
+  <% drones.forEach((aDrone) => { %>
+    <li><%= aDrone.droneName %></li>
```

Comment:
👎 you needed to render not only the name but the propellers and max speed as well.